src/contracts/Protocol/QuestionAPI.sol

```diff
@@ -115,13 +115,17 @@ contract QuestionAPI is Ownable, NFTLocked, FunctionLocked {
      * @param claimLimit The limit for the amount of people that can claim the challenge
      * @return questionId The question id
      */
-    function createChallenge(string calldata uri, uint256 claimLimit) public onlyHolder(PROGRAM_MANAGER_ROLE) returns (uint256) {
+    function createChallenge(
+        string calldata uri,
+        uint256 claimLimit,
```

Comment:
can we make these numbers smaller?  Also should we set a max claimLimit or max threshold?
also can you add param documentation for threshold above function

Reply:
Is there any reason to make the numbers smaller? And we do set both claimLimit and threshold in `        _claimController.initializeQuestion(questionId, claimLimit, threshold);
`

Reply:
meaning do they need to be unit256? not a biggie just a question

Reply:
Ah yes fair, my understanding was that uint256 is more gas friendly. Uint8 does uint8(uint256(1)), therefore costing more gas.